automatic/tor-browser/tools/chocolateyInstall.ps1

```diff
@@ -8,10 +8,10 @@ $destinationFolder = GetInstallDirectory -toolsPath $toolsDir
 $packageArgs = @{
   PackageName  = 'tor-browser'
   FileType     = 'exe'
-  Url          = 'https://www.torproject.org/dist/torbrowser/12.5.6/torbrowser-install-12.5.6_ALL.exe'
-  Url64        = 'https://www.torproject.org/dist/torbrowser/12.5.6/torbrowser-install-win64-12.5.6_ALL.exe'
-  Checksum     = 'e939c0994619047fabedc43dbfc82be63413ab0a5bc777a4b8ad67d3dde2e974'
-  Checksum64   = '88c33af6f1963eb94683be1fecfada36d17546579098372dcad61ae2137ad7b3'
+  Url          = 'https://dist.torproject.org/torbrowser/13.0.1/tor-browser-windows-x86_64-portable-13.0.1.exe'
+  Url64        = 'https://dist.torproject.org/torbrowser/13.0.1/tor-browser-windows-x86_64-portable-13.0.1.exe'
```

Comment:
This link is already outdated and returns an error 404. As mentioned in #2332 I would suggest using the torproject archive under https://archive.torproject.org/tor-package-archive/torbrowser/ so the future packages won't break every time a new version is released.

Reply:
Thanks @daerSeebaer, I've updated to use the archive links now. It's been hectic so I'm a bit slow on the uptick!